src/main/java/com/netflix/imflibrary/Colorimetry.java

```diff
@@ -22,6 +22,7 @@
     Color6(ColorPrimaries.P3D65,      TransferCharacteristic.SMPTEST2084,       CodingEquation.None),
     Color7(ColorPrimaries.ITU2020,    TransferCharacteristic.SMPTEST2084,       CodingEquation.ITU2020NCL),
     Color_App5_AP0(ColorPrimaries.ACES,    TransferCharacteristic.Linear,       CodingEquation.None),
+    Color8DPP(ColorPrimaries.ITU2020, TransferCharacteristic.ITU2020HLG,        CodingEquation.ITU2020NCL),
```

Comment:
Can we use the name as defined in https://github.com/SMPTE/st2067-21-2016-am1/blob/master/35PM-CD-ST-2067-21-2016-AM1.pdf, i.e. simply Color8?

Reply:
I'll change, it's related to firsts DPP documents which references like that. Made sense to update it.